components/icon-button/style.scss

```diff
@@ -9,6 +9,7 @@
 	position: relative;
 	width: $icon-button-size;	// show only icon on small breakpoints
 	overflow: hidden;
+	text-indent: 4px;
```

Comment:
This enhances the IconButton component so when it has text, the icon indents the text, but it doesn't affect the shape of the IconButton if it has no text.